tests/vmi_configuration_test.go

```diff
@@ -503,10 +507,19 @@ var _ = Describe("[sig-compute]Configurations", func() {
 		})
 
 		Context("with BIOS bootloader method and no disk", func() {
+			withNoDisks := func() libvmi.Option {
+				return func(vmi *v1.VirtualMachineInstance) {
+					vmi.Spec.Domain.Devices.Disks = nil
+					vmi.Spec.Volumes = nil
+				}
+			}
 			It("[test_id:5265]should find no bootable device by default", func() {
 				By("Creating a VMI with no disk and an explicit network interface")
-				vmi := tests.NewRandomVMI()
-				tests.AddExplicitPodNetworkInterface(vmi)
+				vmi := libvmi.NewCirros(
+					withNoDisks(),
```

Comment:
Running Cirros without cloud-init is suppose to be super-slow.
I know it is not related to this change, but could you check if indeed this test takes a long time? If it does, it may worth checking if moving to Alpine helps.

Reply:
Not sure why, but I checked the bootup time, and it doesn't longer than before